README.md

```diff
@@ -388,7 +420,7 @@ Build Date: 2023-10-20T09:45:41+00:00
 
 ```bash
 cd /usr/share/doc/libtdx-attest-dev/examples/
-./test_tdx_attest
+sudo ./test_tdx_attest
```

Comment:
do we really need sudo access ?

Reply:
It appears so:

```
tdx@tdx-guest:/usr/share/doc/libtdx-attest-dev/examples$ ./test_tdx_attest 
		
		TDX report data

 00000000: b8 34 b6 d8 70 8d c4 69 2d 68 8c 56 06 5d b8 1d
 00000010: 2c f3 d1 fd a4 cc 85 1f 49 79 a9 be f4 1d 53 ac
 00000020: 51 09 85 c1 97 49 2b c4 b1 b7 1a b8 14 d2 d5 41
 00000030: c5 a6 3e 6a 72 c3 89 bc 3d 32 7a 31 4f cd de a0

[../tdx_attest.c:480] Permission denied

Failed to get the report
```

Reply:
in the instructions, we log into the guest with root account, how did you end up with tdx user ?

Reply:
I removed all the `sudo`'s from the guest commands.